### PR TITLE
GAZEBO: Initial Gazebo Setup

### PR DESCRIPTION
**Pull request by mattlangford**

### CODE REVIEW

File: mission_control/navigator_launch/launch/gnc.launch [hunk 1]
```diff
@@ -36,10 +36,10 @@
...
         <rosparam param="thruster_FL_cog">[1.5748, 0.6096]</rosparam>
         <rosparam param="thruster_FR_cog">[1.5748, -0.6096]</rosparam>
-        <rosparam param="thruster_BL_theta">45</rosparam>
-        <rosparam param="thruster_BR_theta">-45</rosparam>
-        <rosparam param="thruster_FL_theta">-45</rosparam>
-        <rosparam param="thruster_FR_theta">45</rosparam>
+        <rosparam param="thruster_BL_theta">0.785398</rosparam> <!-- +/- 45 degrees -->
+        <rosparam param="thruster_BR_theta">-0.785398</rosparam>
+        <rosparam param="thruster_FL_theta">-0.785398</rosparam>
+        <rosparam param="thruster_FR_theta">0.785398</rosparam>
```
Comment (DSsoto) [inline mission_control/navigator_launch/launch/gnc.launch]: Good, radians!

File: mission_control/navigator_launch/launch/simulation.launch [hunk 2]
```diff
@@ -1,29 +1,38 @@
...
-	<include file="$(find navigator_launch)/launch/perception.launch"/>
-	<node name="rqt_gui" pkg="rqt_gui" type="rqt_gui"/>
+    <include file="$(find navigator_launch)/launch/gnc.launch"/>
+    <include file="$(find navigator_launch)/launch/perception.launch"/>
+
+    <include file="$(find navigator_gazebo)/launch/goose.launch" if="$(arg gazebo)">
+    	<arg name="gui" value="$(arg gui)"/>
+    </include>
+    
+    <group unless="$(arg gazebo)">
```
Comment (DSsoto) [inline mission_control/navigator_launch/launch/simulation.launch]: Does this just mean "if not <gazebo>"?

File: simulation/navigator_gazebo/include/navigator_gazebo/navigator_thrusters.hpp [hunk 3]
```diff
@@ -0,0 +1,79 @@
+#pragma once
```
Comment (DSsoto) [inline simulation/navigator_gazebo/include/navigator_gazebo/navigator_thrusters.hpp]: Why are you using pragma once for some and include guards for others?

File: simulation/navigator_gazebo/models/wamv/wamv.sdf [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    <link name="base_link">
+      <!-- TODO: Tune these -->
+      <velocity_decay>
+        <linear>0.005</linear>
+        <angular>0.001</angular>
+      </velocity_decay>
+      <inertial>
+        <mass>250</mass>
+        <inertia>
+          <ixx>900.0</ixx>
```
Comment (DSsoto) [inline simulation/navigator_gazebo/models/wamv/wamv.sdf]: Might want to add a comment saying that this is a constant density sphere approximation.

**Comment by mattlangford** [comment]
Just got CI working - realized there was a build error. I have fixed it and will have a new PR tonight.

File: mission_control/navigator_launch/launch/simulation.launch (hunk 2 above)
Comment (mattlangford) [inline mission_control/navigator_launch/launch/simulation.launch]: Yep

**Pull request merged**